pages/Aluminum-Dust.md

```diff
@@ -0,0 +1,27 @@
+Aluminum Dust is one of the nine [dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts) added in Slimefun.
+
+## Obtaining
+
+Aluminum Dust has a ~11.11% chance of being extracted from [Sifted Ore](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Sifted-Ore), by processing it through an [Ore Washer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ore-Washer) or an [Electric Dust Washer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Dust-Washer).
+
+## Usage
+Aluminum Dust can be smelted into [Aluminum Ingots](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Aluminum-Ingot) in a [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or an [Electric Ingot Factory](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Factory).
+
+It is also used in the following [alloys](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ingots#Alloys) for the [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or the [Electric Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Smeltery):
+* [Aluminum Brass Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Aluminum-Brass-Ingot)
+* [Aluminum Bronze Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Aluminum-Bronze-Ingot)
+* [Duralumin Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Duralumin-Ingot)
+
+... and directly in creating other items:
+
+* [Magnet](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magnet)
+* [Duct Tape](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Duct-Tape)
+* [Synthetic Emerald](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Synthetic-Emerald)
+
+Aluminum is one of the most used dusts in Slimefun recipes, either as dust or as [Aluminum Ingot](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Aluminum-Ingot).
+
+## Trivia
+
+* In real life, "aluminium ore/dust" is called "bauxite", which is high content aluminum rock.
```

Comment:
That is not true.

Bauxite is a mineral of high aluminum concentration and the largest **source** of aluminum ore. But is not the same at all, it's a mineral.

Reply:
OK, removed as per the next comment